modules/optimization/doc/content/source/controls/OptimizationMultiAppCommandLineControl.md

```diff
@@ -0,0 +1,24 @@
+# OptimizationMultiAppCommandLineControl.md
+
+!syntax description /Controls/OptimizationMultiAppCommandLineControl
+
+## Overview
+
+This control uses the command line control mechanism to change parameters on the subapp.  This allows us to modify parameters during optimization that are not controllable.  This control is an exact copy of MultiAppCommandLineControl from the stochastic tools module with OptimizeFullSolveMultiApp added as a friend.
```

Comment:
This will need to be fixed with a ticket after the optimization module is merged.  I think STM merged their command line controls into SamplerFullSolveMultiapp and we should do the same thing with OptimizeFullSolveMultiApp.  @GiudGiud Do I open an issue now or after it is merged?

Reply:
Now